.mk/sample.mk

```diff
@@ -2,13 +2,17 @@
 .PHONY: deploy-sample-cr
 deploy-sample-cr:
 	@echo -e "\n==> Deploy sample CR"
-	sed -e 's~:main~:$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl apply -f - || true
+ifeq (main,$(VERSION))
+	kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector.yaml || true
+else
+	kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector_versioned.yaml || true
+endif
```

Comment:
This is what @eranra wrote, on purpose I think .. I'm not against discussing that, but it doesn't really relate to that PR

To bring some context: I did did small change in the makefile because, while writing the doc, I figured out that deploying a "versionned" sample CR wouldn't work as expected, especially as the underlying component images are not tied anymore with the operator version. So now, I recommend switching to the version tag in git before deploying / applying CR

Reply:
Okay, that change is fine.  We can ignore my comment.